fix(docs-infra): various small fixes/improvements

gkalpak opened the pull request:
Various small fixes/improvements related to docs-infra.
See individual commits for details.

petebacondarwin reviewed (commented):
Typo in 2nd commit: `becuase`.

gkalpak committed:
docs: use correct service name when creating error handler in `http` example

gkalpak committed:
fix(docs-infra): avoid prepending version info to error messages twice

Previously, we used to prepend version info to error messages by
updating their `message` property and their `stack` property. However,
due to the fact that on some browsers `error.stack` is lazily computed
based on `error.message` the first time the `stack` property is
accessed, this resulted in prepending the version info twice.

This commit ensures that the version info is only prepended to
`error.stack` once to avoid unnecessary repetition in error messages.

gkalpak committed:
ci: avoid testing angular.io against RxJS v7 twice

Previously, we used to run the angular.io unit tests against both RxJS
v6 (which was the version specified in `package.json`) and RxJS v7 on
CI, as a way to guard against future regressions with RxJS v7.

Now that `package.json` has been updated to RxJS v7, there is no reason
to explicitly re-run the unit test against RxJS v7 on CI, so this commit
removes the corresponding steps.

gkalpak committed:
fix(docs-infra): fix Sass rule resulting in invalid CSS

The way the Sass rule was defined previously resulted in invalid CSS of
the form:
```css
table > thead > { ... }
```

This, in turn, resulted in a build-time warning:
```
Warning:  > styles.9a5a35b070498c08.css:3079:14: warning: Unexpected "{"
    3079 │ table thead > {
         ╵               ^
```

This commit fixes the affected rule (mostly to avoid the warning,
because this change doesn't seem to have any visual difference). It also
fixes other rules that were currently not affected (because they only
had nested rules and no direct styles), but were susceptible to being
affected in the future.

atscott commented:
This PR was merged into the repository by commit 300230c25505e097dc6a8bbfebcea787fbaaca1c.
